root/layout/components/BottomMenu.js

```diff
@@ -251,6 +251,9 @@ const EditingMenu = () => (
       <li>
         <a href="/reports">{l('Reports')}</a>
       </li>
+      <li>
+        <a href="/voting-reports">{l('Voting Reports')}</a>
```

Comment:
Maybe it would be simpler to just replace the above _Vote on Edits_? Both have the same purpose. The new page is just more sophisticated.

Reply:
I guess it might make sense - at the very least we should rename "Vote on Edits" to "All Open Edits" or something, because right now it's actually kind of weird. But I'd be fine with replacing the link too - we might just have to add an "All open edits" link to this page.